multicluster/test/integration/suite_test.go

```diff
@@ -42,7 +44,6 @@ import (
 	antreamcscheme "antrea.io/antrea/multicluster/pkg/client/clientset/versioned/scheme"
 	antreascheme "antrea.io/antrea/pkg/client/clientset/versioned/scheme"
 	"antrea.io/antrea/pkg/signals"
-	//+kubebuilder:scaffold:imports
```

Comment:
Why to move this line? 
I think it was generated automatically before, please help to check if possible to remove it, or it has a specific purpose. Thanks.